src/Features/Core/Portable/PullMemberUp/QuickAction/InterfacePullerWithQuickAction.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeGeneration;
+using Microsoft.CodeAnalysis.Editing;
+using static Microsoft.CodeAnalysis.CodeActions.CodeAction;
+
+namespace Microsoft.CodeAnalysis.PullMemberUp.QuickAction
+{
+    internal class InterfacePullerWithQuickAction : AbstractMemberPullerWithQuickAction
+    {
+        private IMethodSymbol FilterGetterOrSetter(IMethodSymbol getterOrSetter)
```

Comment:
feels like this could just be simplified as `getterOrSetter?.DeclaredAccessibility == Public ? getterOrSetter : null`.

Reply:
True, I need to get myself familiar with ?. operator.